package-lock.json

```diff
@@ -1,8 +1,2161 @@
 {
     "name": "vscode-clangd",
     "version": "0.1.10",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```

Comment:
is this change intentional? Looking at our previous upgrade PR, it didn't change the version, e.g. https://github.com/clangd/vscode-clangd/pull/91/files

Reply:
The change was made automatically when I ran `npm install` after updating the dependency in `package.json`.

In terms of what it means, a quick look at the [docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json) suggests that:

  * npm v7 introduced a new lockfile format version (version 2)
  * version 2 is backwards compatible with version 1

So I think this should be fine.

Reply:
I see, fair enough. Thanks for the explanations!